### DISCUSSION TITLE
I don't want to show up projects I'm ashamed or not involved anymore

### DISCUSSION THREAD

**Discussion by esimov** [community/community#13314] [Other Feature Feedback, Questions, & Ideas]
I would like an option in the profile settings to disable the new feed page. In case this is not to be considered I would like at least the possibility of marking some recommended projects as irrelevant or unwanted to not show up anymore. Also once I click to the following button I would like this to be retained as the default page and when accessing the github next time to not show up the new feed page by default.

**Comment by jfullstackdev** [comment]
yup, you have similar request and as far as I can remember, they are working on it

and

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by esimov** [reply]
No, what I would like is to have an option under the three dots button like "Remove" or "Don't show me anymore".

**Reply by jfullstackdev** [reply]
yup, that's also one common request and as far as I can remember they are working on it already
